config/taxonomy/codegen.spec.yaml

```diff
@@ -20,7 +20,7 @@ paths:
         description: Policy Manager Request Object.
```

Comment:
Instead of having a `codegen.spec.yaml` start creating OpenAPI specs for the different connector types.

Reply:
created policymanager.spec.yaml and catalogconnector.spec.yaml now. Please review. Thanks.